### PR TITLE
[Blazor][Fixes #15155] Clarifies the message to turn on detailed erro…

### LINKED ISSUE #15155: [Blazor] Improve error message to include how to set options on appSettings.Development.json

### PR DESCRIPTION
**Pull request by javiercn**

### CODE REVIEW

**Review by pranavkm — APPROVED** [review]
/cc @danroth27

File: src/Components/Server/src/Circuits/CircuitHost.cs [hunk 1]
```diff
@@ -545,7 +545,7 @@ private string GetClientErrorMessage(Exception exception, string additionalInfor
             else
             {
                 return $"There was an unhandled exception on the current circuit, so this circuit will be terminated. For more details turn on " +
-                    $"detailed exceptions in '{typeof(CircuitOptions).Name}.{nameof(CircuitOptions.DetailedErrors)}'. {additionalInformation}";
+                    $"detailed exceptions by setting 'detailedErrors: true' in 'appSettings.Development.json' or set '{typeof(CircuitOptions).Name}.{nameof(CircuitOptions.DetailedErrors)}'. {additionalInformation}";
```
Comment (danroth27) [inline src/Components/Server/src/Circuits/CircuitHost.cs]: Don't we usually PascalCase config values?

Comment (javiercn) [reply src/Components/Server/src/Circuits/CircuitHost.cs]: no. It's json

Comment (javiercn) [reply src/Components/Server/src/Circuits/CircuitHost.cs]: https://github.com/aspnet/AspNetCore/blob/a4d9faefe2ed6fbebd500413032336607ec92778/src/Hosting/Abstractions/src/WebHostDefaults.cs#L13

Comment (danroth27) [reply src/Components/Server/src/Circuits/CircuitHost.cs]: But look at the appSettings.json files in our templates - they are all PascalCased.

Comment (javiercn) [reply src/Components/Server/src/Circuits/CircuitHost.cs]: updated the casing to make you happy.

**Review by pranavkm — APPROVED** [review]

**Pull request merged**